extensions/shared/blocks/cover/index.js

```diff
@@ -11,15 +11,23 @@ import { addFilter } from '@wordpress/hooks';
  * Internal dependencies
  */
 import coverEditMediaPlaceholder from './cover-media-placeholder';
-import isCurrentUserConnected from '../../is-current-user-connected';
 import jetpackCoverBlockEdit from './edit';
-
+import { isUpgradable } from './utils';
 import './editor.scss';
 
-if ( isCurrentUserConnected() ) {
```

Comment:
I see you remove this checking. Isn't needed? Maybe we can add it to  the `isUpgradable()` function.

Reply:
Since we're only interested in simple sites, I think we can omit that check